packages/react-components/react-shared-contexts/src/MaterialTypeContext/MaterialTypeContext.ts

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+export type MaterialTypeContextValue = MaterialType;
+
+export const MaterialType = {
+  /**
+   * A solid surface that does not allow any light to pass through.
+   */
+  Opaque: 'opaque',
```

Comment:
The design spec does not use this nomenclature. If we want to formalize this in another way let's go through it with designers first.

Reply:
If we invent something that does not align with design language, this can result in design specs that confuse feature engineers